doc/nrf/known_issues.rst

```diff
@@ -1702,7 +1740,22 @@ NCSDK-7982: Partition manager: Incorrect partition size linkage from name confli
 DFU and FOTA
 ============
 
-.. rst-class:: v2-1-2 v2-1-1 v2-1-0 v2-0-2 v2-0-1 v2-0-0 v1-9-2 v1-9-1 v1-9-0 v1-8-0 v1-7-1 v1-7-0
+.. rst-class:: v2-1-2 v2-1-1 v2-1-0
+
+NCSDK-18357: Serial recovery does not work on nRF5340 network core
+  If a network core serial recovery is attempted using MCUboot serial recovery, the upload will complete, but the image will not be transferred to the network core and the upgrade will fail.
+
+.. rst-class:: v2-1-2 v2-1-1 v2-1-0 v2-0-2 v2-0-1 v2-0-0
+
+NCSDK-18422: Serial recovery fails to write to slots in QSPI
```

Comment:
Hi @nordicjm,
Cherry-picking the `11c5149a7dd6c9f7581e971656bae461c135e31a` into sdk-muboot could be a workaround here, right?